src/components/ProfileImage.jsx

```diff
@@ -0,0 +1,13 @@
+function ProfileImage({ image }) {
+    return ( 
+        <img
+        src={image}
+        className="profile"
+        alt="profile"
+      />
```

Comment:
```suggestion
          src={image}
          className="profile"
          alt="profile"
        />
```
Añademe la tabulacion hombre! :D